### PR TITLE
replaces old freelancer sprites and changes a few things of their loadout

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
gives freelancers new sprites, and unique coifs.
Changes the upp belt to the normal one.

## Why It's Good For The Game
sprites in the style good.
image

## Changelog
:cl:
add: Added new freelancer sprites.
del: Removed freelancer sprites, git will remember them. Nobody else will.
fix: Freelancers no longer use the upp belt. They use the normal medical belt.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/breath.dm [hunk 1]
```diff
@@ -66,6 +66,12 @@
 	flags_cold_protection = HEAD
 	min_cold_protection_temperature = ICE_PLANET_MIN_COLD_PROTECTION_TEMPERATURE
 
+/obj/item/clothing/mask/rebreather/scarf/freelancer
+	name = "heat absorbent coif"
```
Comment (XSlayer300) [inline code/modules/clothing/masks/breath.dm:70]:
```suggestion

```
Child of above, name will be the same.

File: code/modules/clothing/masks/breath.dm [hunk 2]
```diff
@@ -66,6 +66,12 @@
...
+	desc = "A close-fitting cap that covers the top, back, and sides of the head. Can also be adjusted to cover the lower part of the face so it keeps the user warm in harsh conditions."
```
Comment (XSlayer300) [inline code/modules/clothing/masks/breath.dm:71]:
```suggestion

```
Child of above, name will be the same.

File: code/modules/clothing/masks/breath.dm [hunk 3]
```diff
@@ -66,6 +66,12 @@
...
+	icon_state = "coif"
```
Comment (XSlayer300) [inline code/modules/clothing/masks/breath.dm:72]:
```suggestion

```
Child of above, name will be the same.

**Review by Hughgent — APPROVED** [review]
code is fine

**Review by TheBimmer231 — APPROVED** [review]
Very good

**Pull request merged**